src/common/services/api/donations.service.js

```diff
@@ -101,7 +100,7 @@ function DonationsService( cortexApiService, profileService ) {
     gifts = angular.isArray( gifts ) ? gifts : [gifts];
     return cortexApiService.put( {
       path: ['donations', 'recurring', cortexApiService.scope, 'active'],
-      data: {donations: map( gifts, ( gift ) => omit( gift, ['_selectedGift', 'paymentMethod'] ) )}
+      data: {donations: { 'donation-lines': map(gifts, 'toObject') } }
```

Comment:
This is the line the NPE occurred on:
`if (DonationStatus.SUSPENDED.getStatus().equals(donationLine.getDonationLineStatus()))`